ai/discord.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: "Discord bot"
+description: "Add a bot to your Discord server that answers questions based on your documentation."
+keywords: ["Discord", "bot", "Q&A", "customer support"]
+---
+
+<Info>
+  Discord integrations are available on [Pro and Custom plans](https://mintlify.com/pricing?ref=discord) with access to the Mintlify assistant.
+</Info>
+
+The Discord bot supports your community with real-time answers from your documentation. The bot is trained on your documentation, so it is always up-to-date and answers questions by directly citing your docs.
+
+The Discord bot responds to questions when a user mentions it with the `@` symbol.
```

Comment:
Can you wordsmith this?

```suggestion
The Discord bot responds to questions when a user mentions it with the `@` symbol or it will auto-respond in an #ask-ai channel.
```